radar-commons-server/src/main/avro/kafka/aggregate_list_state.avsc

```diff
@@ -0,0 +1,9 @@
+{
+    "namespace": "org.radarcns.stream.collector",
```

Comment:
Is there any reason we keep these schemas in radar-commons instead of radar-schemas?

Reply:
Hmm yes good point. I can start migrating this component indeed...